test/test_utils.c

```diff
@@ -4,6 +4,7 @@
 #include "options/path.h"
 #include "osdep/subprocess.h"
 #include "test_utils.h"
+#include <common/msg.h>
```

Comment:
huh?

Reply:
For `mp_log_new`

Reply:
Shouldn't it be `"common/msg.h"` and alphabetized though

Reply:
changed